portal-frontend/src/app/shared/file-drag-drop/file-drag-drop.component.scss

```diff
@@ -0,0 +1,30 @@
+@import "../../../styles/colors";
+
+.file-input {
+  display: none;
+}
+
+.file-drag-drop {
+  border: 2px dashed rgba($grey-dark, 0.5);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.icon {
+  font-size: 80px;
+  color: $grey-dark;
+}
+
+.drag-text {
+  color: $grey-dark;
+  font-size: 16px;
```

Comment:
Should this be em?

Reply:
I'm not sure, I'm just following Sandra's designs